config/routes.rb

```diff
@@ -16,7 +16,7 @@
   get '(/:year)(/:month)(/:day)/page(/1)', to: redirect { |_, req|
```

Comment:
@veganstraightedge when you tested locally, did you edit line 16 *and* line 19?

I only edited 19 to make the tests pass...

Reply:
I only edited `19` in my testing.

Reply:
ok, cool.

i clicked around on the review app and it seems good

at least if we get bug reports about broken links we know where to look